sdk/keyvault/keyvault/test/main.c

```diff
@@ -1,18 +1,45 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
+#include <az_client_secret_credential.h>
 #include <az_keyvault.h>
 
 #include <_az_cfg.h>
 
 int exit_code = 0;
 
 int main() {
-  // Creates keyvault client and init it
+  // Creates keyvault client
   az_keyvault_keys_client client;
-  az_keyvault_keys_client_options options;
-  az_span uri = { 0 };
 
-  az_result operation_result = az_keyvault_keys_client_init(&client, uri, &options);
+  // create credentials as client_id type
+  az_client_secret_credential auth = { 0 };
+  // Get secrets from ENV
+  char * const TENANT_ID = getenv("tenant_id");
+  char * const CLIENT_ID = getenv("client_id");
+  char * const CLIENT_SECRET = getenv("client_secret");
+  // init auth_credentials struc
+  az_result const creds_retcode = az_client_secret_credential_init(
+      &auth,
+      (az_span){ .begin = TENANT_ID, .size = strlen(TENANT_ID) },
```

Comment:
@sergey-shandar , should we write a function to transform run-time zero-terminated string into a span?